content/docs/intro/languages/dotnet.md

```diff
@@ -102,15 +97,12 @@ To deploy your infrastructure run `pulumi up` and the Pulumi engine automaticall
 
 This `csharp` template is cloud agnostic, and you will need to install NuGet packages for the cloud provider of your choice. Additional templates are available:
 
-* `aws-csharp`: a starter AWS project in C#
-* `azure-csharp`: a starter Azure project in C#
-* `gcp-csharp`: a starter GCP project in C#
-{{% /md %}}
-</span>
+* `pulumi new aws-csharp`: creates a starter AWS C# project
+* `pulumi new azure-csharp`: creates a starter Azure C# project
+* `pulumi new gcp-csharp`: creates a starter Google Cloud C# project
```

Comment:
Just to make it a little more clear for people skimming the page that these templates are to be used with `pulumi new`.

Reply:
LGTM.